src/home/Home.jsx

```diff
@@ -1,8 +1,10 @@
 import React from "react";
+import Footer from "./Footer";
 import './Home.css';
 export default function Home() {
 
     return (
+        <div>
```

Comment:
Please add proper indentation to the jsx.

Reply:
Ok @Rahulm2310 👍